saleor/payment/gateways/adyen/plugin.py

```diff
@@ -0,0 +1,512 @@
+        {"name": "Automatic payment capture", "value": False},
+        {"name": "HMAC secret key", "value": ""},
+        {"name": "Notification user", "value": ""},
+        {"name": "Notification password", "value": ""},
+    ]
+
+    CONFIG_STRUCTURE = {
+        "API key": {
+            "type": ConfigurationTypeField.SECRET,
+            "help_text": (
+                "To submit payments to Adyen, you'll be making API requests that are "
+                "authenticated with an API key. You can generate API keys on your "
+                "Customer Area."
+            ),
+            "label": "API key",
+        },
+        "Merchant Account": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": "Yout merchant account name.",
+            "label": "Merchant Account",
+        },
+        "Supported currencies": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": "Determines currencies supported by gateway."
+            " Please enter currency codes separated by a comma.",
+            "label": "Supported currencies",
+        },
+        "Client Key": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": (
+                "The client key is a public key that uniquely identifies a web service "
+                "user. Each web service user has a list of allowed origins, or domains "
+                "from which we expect to get your requests. We make sure data cannot "
+                "be accessed by unknown parties by using Cross-Origin Resource Sharing."
+                "Not required for Android or iOS app."
+            ),
+            "label": "Client Key",
+        },
+        "Origin Url": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": (
+                "The origin URL of the page where you are rendering the Drop-in. This "
+                "should not include subdirectories and a trailing slash. For example, "
+                "if you are rendering the Drop-in on "
+                "https://your-company.com/checkout/payment, specify here: "
+                "https://your-company.com. Not required for Android or iOS app."
+            ),
+            "label": "Origin Url",
+        },
+        "Live": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": (
+                "Leave it blank when you want to use test env. To communicate with the"
+                " Adyen API you should submit HTTP POST requests to corresponding "
+                "endpoints. These endpoints differ for test and live accounts, and also"
+                " depend on the data format (SOAP, JSON, or FORM) you use to submit "
+                "data to the Adyen payments platform. "
+                "https://docs.adyen.com/development-resources/live-endpoints"
+            ),
+            "label": "Live",
+        },
+        "Automatically mark payment as a capture": {
```

Comment:
@patrys Please review these names and descriptions.

Reply:
Shouldn't this be something like `adyen-auto-capture` instead of an English sentence?